dbt/include/databricks/macros/materializations/table.sql

```diff
@@ -29,8 +29,9 @@
 
   {% set should_revoke = should_revoke(old_relation, full_refresh_mode=True) %}
   {% do apply_grants(target_relation, grant_config, should_revoke) %}
-  {% do apply_tblproperties_python(target_relation, tblproperties, language) %}
-
+  {% if language=="python" %}
+    {% do apply_tblproperties(target_relation, tblproperties) %}
```

Comment:
Previously we only used alter statements to apply tblproperties to python models.  Now we reuse this macro for other things.